woocommerce/woocommerce-admin-test-helper
# Add features tab

Adds a feature tab that allows us to more easily enable and disable features that are in development.

### Testing instructions

- Load this branch and enable the plugin
- Go to **Tools > WC Test Helper > Features**
- It should list all the available features
- Toggle one of the features like `analytics` for example, the value in the table should update
- Refresh the page, that feature should now be disabled (Analytics should not be available)
- Go the the Features helper page again
- Notice how the `Reset to defaults` should be enabled, click this
- It should disable and the value in the features table be updated to it's default
- Refresh the page, and the previously toggle feature should be default again (Analytics should show again)
